canonical/lxd-ui
# feat: [WD-14036] Duplicate instance (lxc copy)

## Done

- Added DuplicateInstance Button and Form
- Added duplicate instance API call.

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, [build and run as described in the docs](../CONTRIBUTING.md#setting-up-for-development).
2. Perform the following QA steps:
    - Navigate to an instance
    - Select "Duplicate instance" in header.
    - Fill out form and select "Duplicate & Create"
    - Verify that another instance has been created with the same details.

## Screenshots

![image](https://github.com/user-attachments/assets/47938d7b-cb39-45ce-81ce-d79a91ce1b26)
